go/vt/srvtopo/resilient_server.go

```diff
@@ -308,7 +373,7 @@ func (server *ResilientServer) GetSrvKeyspace(ctx context.Context, cell, keyspac
 	return entry.value, entry.lastError
 }
 
-var watchSrvVSchemaSleepTime = 5 * time.Second
+var watchSrvVSchemaSleepTime = sync2.NewAtomicDuration(5 * time.Second)
```

Comment:
You made that change because the test was giving you a warning in the unit_race?
If it's just for the test, I would change the line that sets the value to be much earlier in the test, before it's used, so you don't need this.

Reply:
Yeah exactly -- for some reason I just tried reverting that change to reproduce it and it doesn't happen any more. Yay races.

Regardless your idea is way better so I just did that.